src/EnergyPlus/VariableSpeedCoils.cc

```diff
@@ -49,7 +49,6 @@
 
 // ObjexxFCL Headers
 #include <ObjexxFCL/Array.functions.hh>
-#include <ObjexxFCL/Optional.hh>
```

Comment:
Woo!  Someone needs to fill in the blank  "Every time we remove an ObjexxFCL::Optional, a(n) _________________"